cloudflare-ddns

```diff
@@ -34,10 +35,26 @@ def load_arguments():
     parser.add_argument('--configure', action='store_true', help='Interactively configure the account and domain for the DDNS updates.')
     parser.add_argument('--update-now', action='store_true', help='Update DNS records right now.')
     parser.add_argument('--debug', action='store_true', help='Print detailed debug output.')
+    parser.add_argument('--silent', action='store_true', help='Do not print any output.')
+    parser.add_argument('--silent-unchanged', action='store_true', help='Only print DNS changes / warning / errors.')
```

Comment:
Per the discussion in the other issue I would recommend removing this option entirely since it doesn't add much value; it's not really "silent" and just feels like a raised log level.

Reply:
Well, request at Issue #47 is:

> Running once a day at midnight with local mail results in the same repeated message of "no changes", but with a quiet option the command would only produce output (and hence an email) if a change was made.

That sounds like my `--silent-unchanged` flag. 
Why not offer the option if someone wants it?